packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -191,7 +175,14 @@ function NavigationMenuItemEdit( {
 					'is-selected': isSelected,
 				} ) }
 			>
-				{ content }
+				<RichText
+					ref={ plainTextRef }
+					className="wp-block-navigation-menu-item__field"
```

Comment:
This class name doesn't make sense to me (field).

Reply:
`wp-block-navigation-menu-item__content` ?